pyomo/solvers/plugins/solvers/CPLEX.py

```diff
@@ -185,6 +185,17 @@ def _presolve(self, *args, **kwds):
         self._warm_start_file_name = kwds.pop('warmstart_file', None)
         user_warmstart = False
         if self._warm_start_file_name is not None:
+            if ' ' in self._warm_start_file_name:
+                if self.version()[:2] >= (12,8):
+                    if '"' not in self._warm_start_file_name:
+                        self._warm_start_file_name \
+                            = '"'+self._warm_start_file_name+'"'
+                else:
+                    raise ValueError(
+                        "Space detected in CPLEX warm start file path/name and "
+                        "CPLEX older than version 12.8.  Please either upgrade "
+                        "CPLEX or remove the space from the warm start path.")
+
```

Comment:
Do quotes work in Linux, or is this just windows? Linux needs escapes, right?

Reply:
The original fix was developed by @ghackebeil, I assume in Linux. I don’t have access to 12.8, so cannot test the quotes.